db-client/src/main/java/com/splicemachine/db/client/net/NetConnectionReply.java

```diff
@@ -296,8 +289,9 @@ private void parseSECCHKreply(NetConnection netConnection) throws DisconnectExce
             // incorrect but consider it a conversation protocol error
             // 0x03 - OBJDSS sent when not allowed.
             //parseSECTKN (true);
-            boolean done = false;
-            byte[] bytes = parseSECTKN(false);
+            //boolean done = false;
+            //byte[] bytes = parseSECTKN(false);
+            parseSECTKN(false);
```

Comment:
this call is not assign to any variable

Reply:
(spotbug) It simply moves the parser forward. We don't do anything with the result.